Tests/FunctionalTests/helpermethod.ts

```diff
@@ -139,6 +139,29 @@ export async function ClickWaitRetry(page: Page, buttonToClick: string, selector
   }
 }
 
+export async function ClickWaitRetryWarnings(page: Page, buttonToClick: string, selectorToWaitFor: string, warningVal: string,
+  timeout: number = 30000, step: number = 1000){
+const maxtime = Date.now() + timeout;
+let success = false;
+
+while (Date.now() < maxtime && !success)
+{
+await page.click(buttonToClick);
+if (await page.locator(selectorToWaitFor).textContent()==warningVal)
```

Comment:
Should this be `===`?
```suggestion
if (await page.locator(selectorToWaitFor).textContent()===warningVal)
```

Reply:
Resolve. Thanks